content/issues/2022-07-01-team-on-vacation.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Summer is here ☀️'
+date: '2022-07-04T08:00:00+00:00'
+resolved: false
+resolvedWhen: '2022-07-08T18:00:00+00:00'
```

Comment:
The README says
```
New, not yet resolved issue or announcement about future maintenance
  * leave resolvedWhen: unset
```
Would it make sense to update it to something like
```
  * set resolvedWhen:
    * to date/time of the expected end of the event or leave it `unset` if you don't know it yet
```

Reply:
Ok! I can't set it to resolved so it makes sense to not put there a resolved date.

Reply:
For me, both are fine because the resolved date is not shown anywhere if `resolved: false`.
So let's leave it as it is now :-)